### PR TITLE
Updated to pep8 and fix path in visualize-force-graph.md

### PR DESCRIPTION
**Pull request by GrantXie**

### CODE REVIEW

File: kgtk/visualize/visualize_api.py [hunk 1]
```diff
@@ -750,4 +728,21 @@ def execute(self):
                 node_categorical_scale=self.node_categorical_scale,
                 edge_categorical_scale=self.edge_categorical_scale,
                 node_gradient_scale=self.node_gradient_scale,
-                edge_gradient_scale=self.edge_gradient_scale)
+                edge_gradient_scale=self.edge_gradient_scale,
+                kwargs=self.kwargs)
+
+
+'''
```
Comment (saggu) [inline kgtk/visualize/visualize_api.py:735]: remove this commented out code/unused string

File: kgtk/visualize/visualize_api.py [hunk 2]
```diff
@@ -664,11 +640,12 @@ def __init__(
             show_text_limit: int = 500,
             node_border_color: str = None,
             tooltip_column: str = None,
-            text_node: str = None,
+            text_node: str = 'false',
```
Comment (saggu) [inline kgtk/visualize/visualize_api.py:643]: again, instead of using string false, why not use boolean `False`

**Review by saggu — CHANGES_REQUESTED** [review]
@GrantXie changes required

File: kgtk/visualize/visualize_api.py (hunk 2 above)
Comment (GrantXie) [reply kgtk/visualize/visualize_api.py]: changed to boolean False

File: kgtk/visualize/visualize_api.py (hunk 1 above)
Comment (GrantXie) [reply kgtk/visualize/visualize_api.py]: removed the commented part

**Pull request merged**